### PR TITLE
Add test for creating service from template

### PR DESCRIPTION
**Pull request by psschwei**
Signed-off-by: Paul S. Schweigert <paulschw@us.ibm.com>

# Changes

Adding test for #214 

/assign @maximilien

### CODE REVIEW

File: pkg/command/service/generate_test.go [hunk 1]
```diff
@@ -176,4 +178,61 @@ func TestNewServiceGenerateCommand(t *testing.T) {
...
+
+		if err := os.WriteFile(templatePath, []byte(templateYaml), 0700); err != nil {
+			t.Fatal(err)
+		}
+
+		cmd := NewServiceGenerateCommand(p)
+		_, err := testutil.ExecuteCommand(cmd, "-n", "1", "-b", "10", "-i", "10", "--namespace", "test-kperf-1", "--template", templatePath)
+		assert.NilError(t, err)
+
+		ksvcClient, _ := p.NewServingClient()
```
Comment (maximilien) [inline pkg/command/service/generate_test.go:227]: Why are you ignoring errors in the next two statements? Might as well check and fail if error

**Review by maximilien — COMMENTED** [review]
Left a comment and also `govet` has some suggestions as well. Please revise.

File: pkg/command/service/generate_test.go (hunk 1 above)
Comment (maximilien) [reply pkg/command/service/generate_test.go:227]: /ok-to-test

Comment (psschwei) [reply pkg/command/service/generate_test.go:227]: I c&p'ed from the `generate service as expected with namespace flag` test... I'll do a fix for both

**Review by maximilien — COMMENTED** [review]
Can you address govet issues?

**Comment by psschwei** [comment]
> Can you address govet issues?

I don't see any... only failure I see is the codecov one

**Review by maximilien — APPROVED** [review]
/approve
/lgtm

Thanks

**Pull request merged**